crates/python/qcs_sdk/qpu/isa.pyi

```diff
@@ -0,0 +1,311 @@
+from enum import Enum
+from typing import List, Optional
+
+from .client import QcsClient
+
+class QcsIsaSerializationError(ValueError):
+    """Raised when an `InstructionSetArchitecture` is not able to be serialized from a given json string."""
+    ...
+
+
+class QcsIsaError(RuntimeError):
+    """Raised when there is a problem fetching `InstructionSetArchitecture` from the QCS API."""
+    ...
+
+
+class Family(Enum):
+    """
+    The architecture family identifier of an `InstructionSetArchitecture`.
+
+    Value "Full"  implies that each node is connected to every other (fully-connected architecture).
+    """
+
+    Full = "Full",
+    Aspen = "Aspen",
+    Ankaa = "Ankaa",
+
+
+class Node:
```

Comment:
Did you have to write this all up by hand?

Reply:
Lot of copy pasting, with some polishing